vm/src/format.rs

```diff
@@ -427,6 +427,14 @@ impl FormatSpec {
         self.format_sign_and_align(&magnitude_string, sign_str)
     }
 
+    #[inline]
+    fn format_int_internal(&self, magnitude: BigInt, radix: u32) -> Result<String, &'static str> {
```

Comment:
```suggestion
    fn format_int_radix(&self, magnitude: BigInt, radix: u32) -> Result<String, &'static str> {
```
when we can name it specific name, I like to avoid too common names like `internal`